aleph/validation/schema/entity.yml

```diff
@@ -137,3 +137,58 @@ EntityTag:
     count:
       type: integer
       example: 1
+
+EntityExpandStats:
+  type: object
+  properties:
+    property:
+      type: object
+    count:
+      type: integer
+
+EntityGraph:
+  type: object
+  properties:
+    nodes:
+      type: array
+      items:
+        $ref: '#/components/schemas/EntityGraphNode'
+    edges:
+      type: array
+      items:
+        $ref: '#/components/schemas/EntityGraphEdge'
```

Comment:
Maybe the response type to this should be some extended form of  EntityExpandStats, where things are grouped by property? Otherwise a straight list of entities could also work...